Library/Formula/meanwhile.rb

```diff
@@ -0,0 +1,301 @@
+class Meanwhile < Formula
+  desc "Implementation of Lotus Sametime Community Client protocol"
+  homepage "http://meanwhile.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/meanwhile/meanwhile/1.0.2/meanwhile-1.0.2.tar.gz"
+  sha256 "38a8059eaa549cbcb45ca0a5b453e9608ceadab360db2ae14581fb22ddabdf23"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  # glib has been updated:
+  #   error "Only <glib.h> can be included directly."
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!